__tests__/tooling/__fixtures__/lib/json-schema.js

```diff
@@ -67,6 +67,14 @@ function buildSchemaDefault(opts) {
     props.default = opts.default ? opts.default : false;
```

Comment:
This entire fixture file is still a huge mess and I want to at some point overhaul it into a general OAS fixture library but alas time.